hack/kind-cluster-build.sh

```diff
@@ -132,7 +132,8 @@ done
 
 echo "start to create k8s cluster"
 kind create cluster --config ${configFile} --image kindest/node:${k8sVersion} --name=${clusterName}
-export KUBECONFIG="$(kind get kubeconfig-path --name=${clusterName})"
+kind get kubeconfig --name=${clusterName} > ${workDir}/config
```

Comment:
use an env variable here?

Reply:
Seems it's not able to get the kubeconfig path directory from kind 0.7.0, so I have to dump it to a file and set env variable after that. Do you mean by use env variable directly, how can we do it?

Reply:
sorry, I mean ${workDir}/config, it’s also used in another place.

Reply:
Resolved, @mahjonp PTAL again.